pkg/graph/schema.graphql

```diff
@@ -53,6 +53,37 @@ type CedarSystem {
 	systemMaintainerOrgComp: String
 }
 
+"""
+Represents a cedar system that a user has bookmarked in the system repository
+"""
+type CedarSystemBookmark {
+  euaUserId: String!
+  cedarSystemId: UUID!
+}
+
+"""
+The data needed to bookmark a cedar system
+"""
+input CreateCedarSystemBookmarkInput {
+  euaUserId: String!
```

Comment:
Do we need `euaUserId` as part of the input if the current user (from the Context object) is whats used to pull the EUA ID?

```go
EUAUserID: appcontext.Principal(ctx).ID()
```

(from your `CreateCedarSystemBookmark` later in this PR)